test/runcitests.cmd

```diff
@@ -127,10 +130,13 @@ set _HadFailures=0
 
   call :do %_TestDir%\runnativetests.cmd -%1%2 > %_TestDir%\logs\%1_%2\nativetests.log 2>&1
 
-  if ERRORLEVEL 1 set _HadFailures=1
+  if %ERRORLEVEL% NEQ 0 (
+    echo -- runcitests.cmd ^>^> runnativetests.cmd failed

```

Comment:
AFAIK right now it doesn't contain any information that isn't already echoed to stdout.

Also, we already print out that we're xcopying E:\A\_work\5\s\test\logs\x86_test\nativetests.log (among other things) to the testlogs folder (which is then copied to the drop path). Making the message more specific would probably get too specific to the VSO build infra (which is bad since this can be used for local builds as well).

I think this is probably unnecessary.